internal/pkg/cli/env_init.go

```diff
@@ -189,6 +178,43 @@ func (o *initEnvOpts) delegateDNSFromProject(project *archer.Project) error {
 	return nil
 }
 
+func (o *initEnvOpts) askEnvName() error {
+	if o.EnvName != "" {
+		return nil
+	}
+
+	envName, err := o.prompt.Get(envInitNamePrompt, envInitNameHelpPrompt, validateEnvironmentName)
+	if err != nil {
+		return fmt.Errorf("prompt to get environment name: %w", err)
+	}
+	o.EnvName = envName
+	return nil
+}
+
+func (o *initEnvOpts) askEnvProfile() error {
+	if o.EnvProfile != "" {
+		return nil
+	}
+
+	names, err := o.sessProvider.ProfileNames()
+	if err != nil {
+		return fmt.Errorf("retrieve profile names: %w", err)
+	}
+	if len(names) == 0 {
+		return errNamedProfilesNotFound
```

Comment:
Wonder if we always want to return a predefined custom error type when the items array to be selected is empty. It seems like the behaviors are not consistent across the `cli` package.

Reply:
mm good question, if we pass an empty list to the `prompt` library then it immediately errors with a message that's not really helpful.  
So yeah, I'd say we should be always validating that the list is not empty to provide a more useful message.